### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by allygator**

### CODE REVIEW

**Comment by allygator** [comment]
This does not have the correct path for data travel at the moment. This just gets the login page and context setup

File: src/components/Login.js [hunk 1]
```diff
@@ -1,8 +1,127 @@
...
+    firebase.doSignInWithEmailAndPassword(email, pass).catch(error => {
+      console.log(error);
+    });
+  }
+
+  function signup(email, pass) {
+    firebase
+      .doCreateUserWithEmailAndPassword(email, pass)
+      .then(authUser => {
+        console.log(userData);
```
Comment (ToanQVu) [inline src/components/Login.js:37]: console.log() statements should be deleted

Comment (allygator) [reply src/components/Login.js:37]: This is important to leave in until its published to master as its needed for verifying database access and flow

**Review by ToanQVu — APPROVED** [review]
It is kinda working.

**Pull request merged**